canela/bin/soap.py

```diff
@@ -87,7 +89,9 @@ def create_soap(filepath, nmax, lmax, rcut, procs):
     print('ALL DONE')
 
 
-# if __name__ == '__main__':
-#     create_soap()
 def main():
-    create_soap()
+    # print version if that is only arg
+    if len(sys.argv) > 1 and sys.argv[1] == '--version':
+        print('soap, ' + __version__)
```

Comment:
I used this approach for argparse, but there's an easier way with click:
You can add the following decorator to your script function:
@click.version_option(__version__)